core/tool_roundtrip.c

```diff
@@ -26,6 +26,9 @@ License along with this library.
 #include "oio_core.h"
 #include "oio_sds.h"
 
+#define STRING1 "toto"
```

Comment:
Let's manage random strings (cf. `oio_str_randomize`) instead of hand-coded ones.